modules/simplytest_submission/src/SubmissionAccessControlHandler.php

```diff
@@ -43,7 +43,7 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
   protected function checkCreateAccess(AccountInterface $account, array $context, $entity_bundle = NULL) {
     $result = parent::checkCreateAccess($account, $context, $entity_bundle);
     if ($result->isNeutral()) {
-      return AccessResult::allowedIfHasPermission($account, 'add submission entities');
```

Comment:
Why remove this? (I'm not sure why to keep it either, just asking out of interest)